packages/yoroi-extension/chrome/extension/ergo-connector/api.js

```diff
@@ -35,7 +35,11 @@ import JSONBigInt from 'json-bigint';
 import { BIP32PrivateKey, } from '../../../app/api/common/lib/crypto/keys/keyRepository';
```

Comment:
Confirmed the disabling of the ergo connector in the same version as this will be released. There will be a separate task to disable it, this PR can then pull and be updated

Reply:
Not removing the connector code fully yet, the take-down of Ergo is happening in multiple steps